jdi-light/src/main/java/com/epam/jdi/light/elements/complex/WebList.java

```diff
@@ -58,10 +60,11 @@ public WebList setCore(JDIBase base) {
         super.setCore(base);
         return this;
     }
-    public WebList() { elements.useCache(false); setTextType(SMART_LIST); noValidation(); }
-    public WebList(By locator) { this(); setLocator(locator);}
+    public WebList() { this.jdiIndex = 1; elements.useCache(false); setTextType(SMART_LIST); noValidation(); }
+    public WebList(int jdiIndex) { this.jdiIndex = jdiIndex; elements.useCache(false); setTextType(SMART_LIST); noValidation(); }
+    public WebList(By locator, int jdiIndex) { this(jdiIndex); setLocator(locator);}
```

Comment:
as I said above we need:   public WebList(By locator) { this(1); setLocator(locator);}

Reply:
Originally I wanted to do this in the same way as you said.
But finally desided that program will be less complicated if we reveal this index to other programmers because it's business feature - whether to use 0 or 1. And if users have any necessity to use WebList in the future they can easily control whether to work with default selenium or jdi weblist.